loadgame.py

```diff
@@ -0,0 +1,14 @@
+def loadUserLetters(save):
```

Comment:
Looks like you have all the necessary components here! I wonder if we should also save the puzzle ie. how it looked/the order of the letters being displayed.

Reply:
I guess I don't know if this is already done which is why comments would be very helpful x3

Reply:
Comments have been added. Let me know if there's anything specific you still need.